script.js

```diff
@@ -1,16 +1,105 @@
+    let pilotStatus = document.getElementById('pilotStatus');
+    let copilotStatus = document.getElementById('copilotStatus');
+
+
+
+    // Validate user enters something for every field
+
+    if (pilotName.value.trim() === ""|| copilotName.value.trim() === "" || fuelLevel.value.trim() === "" || cargoMass.value.trim() === "") {
+        alert("All fields are required!");
+        event.preventDefault();
+    }
+
+    //Validate user enters text for names and numbers for fuel and cargo levels
+
+    if (isNaN (pilotName.value)|| isNaN(copilotName.value)) {
+        pilotStatus.innerHTML =`Pilot ${pilotName.value} is ready`;
+        copilotStatus.innerHTML =`Pilot ${copilotName.value} is ready`;
+    }
+    else {
+        alert ("Pilot & Copilot names need to be a text!");
+        event.preventDefault();
+    }
+    if (isNaN (fuelLevel.value)|| isNaN(cargoMass.value)) {
+        alert ("Fuel and cargo levels need to be numbers!");   
+    }
+    else {
+        // If the user submits a fuel level that is too low (less than 10,000 liters), change faultyItems to visible with an updated fuel status stating that there is not enough fuel for the journey. The text of the h2 element, launchStatus, should also change to "Shuttle not ready for launch" and the color should change to red.
+        
+        if(fuelLevel.value < 10000){
+            faultyItems.style.visibility = 'visible';
+            fuelStatus.innerHTML = `Fuel level too low for the journey!`;
+            launchStatus.innerHTML = `Shuttle not ready for launch`;
+            launchStatus.style.color = 'red';
+            event.preventDefault();
+        } else {
+            faultyItems.style.visibility ='visible';
+            fuelStatus,this.innerHTML = 'Fuel level enough for launch';
+        }
+        
+        // If the user submits a cargo mass that is too large (more than 10,000 kilograms), change the list to visible with an updated cargo status stating that there is too much mass for the shuttle to take off. The text of launchStatus should also change to "Shuttle not ready for launch" and the color should change to red.
+        if(cargoMass.value > 10000){  
+            faultyItems.style.visibility = 'visible';
+            cargoStatus.innerHTML = `Too much cargo mass for the shuttle to take off!`;
+            launchStatus.innerHTML = `Shuttle not ready for launch`;
+            launchStatus.style.color = 'red';
+            event.preventDefault();
+         }else{
+            faultyItems.style.visibility = 'visible';
+            cargoStatus.innerHTML = `Shuttle is ready for launch.`;
+         }
+
+         //If the shuttle is ready to launch, change the text of launchStatus to green and display "Shuttle is ready for launch".
+
+        if(fuelLevel.value >= 10000 && cargoMass.value <= 10000) { 
+            faultyItems.style.visibility = 'visible';
+            launchStatus.innerHTML = `Shuttle is ready for launch.`;
+            launchStatus.style.color = 'green';
```

Comment:
```suggestion
            launchStatus.style.color = 'rgb(65, 159, 106)';
```
This is not required to have a passing assignment, but you can update this value to equal rgb(65, 159, 106) to make it pass the unit test. It's more for practice with reading unit tests.